sklearn/cluster/_bicluster.py

```diff
@@ -130,8 +132,8 @@ def fit(self, X, y=None):
         self : object
             SpectralBiclustering instance.
         """
+        self._validate_params()
         X = self._validate_data(X, accept_sparse="csr", dtype=np.float64)
-        self._check_parameters(X.shape[0])
```

Comment:
```
self._check_parameters(X.shape[0])
```
should be kept: this is the call to the parameter validation and calls (among others)  the check on `n_best` and `n_components`

Reply:
Thanks!  I had removed this call due to some confusion with my IDE no longer recognizing this method, but I see that the test passes with it back in :)